sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/MessageState.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.ServiceBus
+{
+    /// <summary>Represents the message state of the <see cref="ServiceBusReceivedMessage"/></summary>
+    public enum MessageState
```

Comment:
I'm guessing we will end up naming this ServiceBusMessageState.

Reply:
Or even ServiceBusReceivedMessageState?

Reply:
Perhaps, but I think ServiceBusMessageState should be descriptive enough and we have several other types that we have named in this way.